external-providers/generic-external-provider/main.go

```diff
@@ -10,13 +10,109 @@ import (
 	"github.com/konveyor/analyzer-lsp/provider"
 	"github.com/konveyor/generic-external-provider/pkg/generic"
 	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
 var (
 	port = flag.Int("port", 0, "Port must be set")
 )
 
+// For debugging
+type EvaluateCall struct {
+	ServiceClient provider.ServiceClient
+	Cap           string
+	ConditionInfo []byte
+}
+
+// For debugging
+func mainDebug() {
+	logrusLog := logrus.New()
+	logrusLog.SetOutput(os.Stdout)
+	logrusLog.SetFormatter(&logrus.TextFormatter{})
+	// need to do research on mapping in logrusr to level here TODO
+	logrusLog.SetLevel(logrus.Level(5))
+
+	log := logrusr.New(logrusLog)
+
+	ctx := context.TODO()
+	client := generic.NewGenericProvider()
+
+	golangSC, err := client.Init(ctx, log, provider.InitConfig{
+		Location: "/home/jonah/Projects/analyzer-lsp/examples/golang",
```

Comment:
This path doesn't look like one we'd want to commit.

Reply:
IMO, Eiher we remove this, or we make this more generic for testing. 

I would think that removing for this PR and then coming back and making it easy to test w/ the generic provider makes sense to me. OR we make this a feature of kantra, and work on it from that the layer? 

IDK just don't know if we want to change this particular main file yet but I could be wrong

Reply:
Whoops, will fix! Is leaving in the `mainDebug()` function a problem as well? I just found all the stuff I put in there extremely useful when I was having problems.